pkg/operator/yugabytedb/create_controller.go

```diff
@@ -331,7 +331,7 @@ func createContainer(cluster *cluster, containerImage string, isTServerStatefuls
 				},
 			},
 			{
-				Name: envPodName,
```

Comment:
Can the `envPodName` const be removed as well, or is it needed elsewhere?

Reply:
Oops, it can be removed as you said. More precisely, there are other users of envPodName, however, these can be replaced by k8sutil.PodNameEnvVar as the following line. I had to erase it the first time, but I put the changes back in by accident. I'll fix it.

The remaining fix is just for yugabytedb. So I'd like to use "[test yugabytedb]" in the next push, if possible.

Reply:
Sounds good thanks!